src/qt/forms/overviewpage.ui

```diff
@@ -923,8 +907,11 @@
                  <bold>true</bold>
                 </font>
                </property>
+               <property name="toolTip">
+                <string>Combined Balances (including unconfirmed and immature coins)</string>
+               </property>
                <property name="text">
-                <string>Combined Balances (including immature coins)</string>
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Combined Balances&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Is the inline html in these string elements (here and elsewhere) really necessary? The raw contents get sent off to transifex for translation and having the html tags embedded in the string can and most likely will cause translator confusion and/or error.

Reply:
Translatability over usability...that's a new one :-)

IMO having **bold** text elements helps to read the online help text, but on the other side I also see the problems with translation (I have translated quite some texts with HTML in the past, though).
Will change it.